jupytercad/notebook/cad_document.py

```diff
@@ -108,5 +427,102 @@ def _get_yobject_index_by_name(self, name: str) -> int:
                     return index
         return -1
 
+    def _new_name(self, obj_type: str) -> str:
+        n = 1
+        name = f"{obj_type} 1"
+        objects = self.objects
+
+        while name in objects:
+            name = f"{obj_type} {n}"
+            n += 1
+
+        return name
+
     def _repr_mimebundle_(self, **kwargs):
         return self.render()
+
+
+class PythonJcadObject(BaseModel):
```

Comment:
I'm not sure if we will have the circular import issue but if not, can we split this file into multiple smaller files?

Reply:
I moved everything in a single file to prevent a circular import indeed.